### PR TITLE
Add link to self-managed backend on self-hosted pages.

### PR DESCRIPTION
**Pull request by clstokes**
Attempt to help with confusion between Self-Hosted and Self-Managed backends.

### CODE REVIEW

**Comment by clstokes** [comment]
> Your site preview for commit [33840d1](https://github.com/pulumi/pulumi-hugo/commit/33840d151242073e3db382af11eb0477c3d6179b) is ready! 🎉
> 
> http://pulumi-hugo-origin-pr-110-33840d15.s3-website.us-west-2.amazonaws.com.

Quick links based on this bucket:
- http://pulumi-hugo-origin-pr-110-33840d15.s3-website.us-west-2.amazonaws.com/docs/guides/self-hosted/
- http://pulumi-hugo-origin-pr-110-33840d15.s3-website.us-west-2.amazonaws.com/docs/guides/self-hosted/console/
- http://pulumi-hugo-origin-pr-110-33840d15.s3-website.us-west-2.amazonaws.com/docs/guides/self-hosted/api/

File: themes/default/content/docs/guides/self-hosted/_index.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ meta_desc: Pulumi Enterprise Edition gives you the option to self-host Pulumi wi
 ---
 
 {{% notes type="info" %}}
-Self-hosting is only available with the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+This page is for Self-Hosting of the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```
Comment (davidwrede) [inline themes/default/content/docs/guides/self-hosted/_index.md:11]: I believe the original sentence here is a bit more clear and succinct (but with a slight tweak).

```suggestion
Self-hosting is only available with **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```

File: themes/default/content/docs/guides/self-hosted/_index.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ meta_desc: Pulumi Enterprise Edition gives you the option to self-host Pulumi wi
...
+
+For the **open source self-managed backends** see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```
Comment (davidwrede) [inline themes/default/content/docs/guides/self-hosted/_index.md:13]:
```suggestion
To manage your state with a self-managed backend, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```

File: themes/default/content/docs/guides/self-hosted/api.md [hunk 3]
```diff
@@ -9,7 +9,9 @@ meta_desc: Pulumi API is one of the components required for self-hosting Pulumi.
 ---
 
 {{% notes type="info" %}}
-Self-hosting is only available with the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+This page is for Self-Hosting of the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+
+For the **open source self-managed backends** see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```
Comment (davidwrede) [inline themes/default/content/docs/guides/self-hosted/api.md:14]:
```suggestion
Self-hosting is only available with **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.

To manage your state with a self-managed backend, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```

File: themes/default/content/docs/guides/self-hosted/console.md [hunk 4]
```diff
@@ -9,7 +9,9 @@ meta_desc: Pulumi Console is one of the components required for self-hosting Pul
 ---
 
 {{% notes type="info" %}}
-Self-hosting is only available with the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+This page is for Self-Hosting of the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+
+For the **open source self-managed backends** see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```
Comment (davidwrede) [inline themes/default/content/docs/guides/self-hosted/console.md:14]:
```suggestion
Self-hosting is only available with **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.

To manage your state with a self-managed backend, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```

**Review by davidwrede — CHANGES_REQUESTED** [review]

**Review by davidwrede — APPROVED** [review]

**Pull request merged**